include/segment_symbols.h

```diff
@@ -32,6 +32,11 @@
 #define SEGMENT_BSS_END(segment) (_ ## segment ## SegmentBssEnd)
 #define SEGMENT_BSS_SIZE(segment) ((uintptr_t)SEGMENT_BSS_END(segment) - (uintptr_t)SEGMENT_BSS_START(segment))
 
+#define ROM_FILE(name) \
+    { (uintptr_t)_##name##SegmentRomStart, (uintptr_t)_##name##SegmentRomEnd }
```

Comment:
What about this instead?
```suggestion
    { (uintptr_t) SEGMENT_ROM_START(name), (uintptr_t)SEGMENT_ROM_END(name) }
```